Sources/BraveWallet/Crypto/Search/SwapTokenSearchView.swift

```diff
@@ -22,7 +22,9 @@ struct SwapTokenSearchView: View {
 
   var body: some View {
     let excludedToken = searchType == .fromToken ? swapTokenStore.selectedToToken : swapTokenStore.selectedFromToken
-    TokenList(tokens: swapTokenStore.allTokens.filter { ($0.symbol != excludedToken?.symbol) && ($0.isErc20 || $0.symbol == network.symbol) }) { token in
+    TokenList(tokens: swapTokenStore.allTokens
+      .filter { ($0.symbol != excludedToken?.symbol) && (!$0.isNft || $0.symbol == network.symbol) }
```

Comment:
i guess here not exclude `$0.isErc20` is ok because those erc20 tokens have been removed from token registry? im not 100% sure. but i believe all user added NFTs will always have `isNft` on.

Reply:
The prior logic here was _including_ ERC20, not excluding 🙂. Now just NFTs excluded as Solana tokens are not ERC20